app/clients/zendesk_sell.py

```diff
@@ -148,7 +148,7 @@ def _generate_note_content(contact: ContactRequest) -> str:
         )
 
     def _send_request(
-        self, method: str, relative_url: str, data: str = None
+        self, method: str, relative_url: str, data: str | None = None
```

Comment:
You can also use `Optional` to do this:
```suggestion
        self, method: str, relative_url: str, data: Optional[str] = None
```